packages/user-mgnt/src/features/verifyUser/handler.ts

```diff
@@ -45,19 +42,15 @@ export default async function verifyUserHandler(
 ) {
   const { mobile, email } = request.payload as IVerifyPayload
 
-  let user: IUserModel | null
-
   if (!email && !mobile) {
     return badRequest()
   }
 
-  if (mobile) {
-    user = await User.findOne({ mobile })
-  } else {
-    user = await User.findOne({
-      emailForNotification: { $regex: `^${email}$`, $options: 'i' }
-    })
-  }
```

Comment:
The regex was probably used to achieve case-insensitive matching, which is unnecessary now since all stored emails are already normalized to lowercase at creation time.